doc/sphinx-guides/source/user/data-exploration/worldmap.rst

```diff
@@ -3,29 +3,146 @@
 WorldMap: Geospatial Data Exploration
```

Comment:
What percentage of users are going to be **viewing** maps versus **creating** them? My guess is that more people will be viewing maps since you don't even need to create a Dataverse account to view a map. You just click the button on the dataset or file page. Given this, before we even talk about creating maps, can we have a short section on viewing maps?